README.md

```diff
@@ -13,11 +13,11 @@ This directory contains the following helm charts.
 
 ## Prerequisites
 
-* Helm 3.5+
+You need the [`helm`](https://helm.sh/docs/intro/install/) command. See [Package manager section in Requirements document](https://scalardb.scalar-labs.com/docs/latest/requirements/#package-manager) for details on supported Helm versions.
 
 ## Supported Kubernetes versions
 
-* 1.30.x, 1.29.x, 1.28.x, 1.27.x, 1.26.x
+See [Kubernetes section in Requirements document](https://scalardb.scalar-labs.com/docs/latest/requirements/#platform) for details on supported Kubernetes versions.
```

Comment:
Similar to my previous comment, I think we'll be adding a Requirements doc to the ScalarDL docs. Should we include a "Platform" section in that doc so that we can link to it here?

Reply:
Yes. I think we should add a `Platform` section in ScalarDL's requirements document and link to it from this README.